Containers/nextcloud/cron.sh

```diff
@@ -1,7 +1,20 @@
 #!/bin/bash
 set -eu
 
+wait_for_cron() {
+    set -x
+    while [ -n "$(pgrep -f /var/www/html/cron.php)" ]; do
+        echo "Waiting for cron to stop..."
+        sleep 5
+    done
+    echo "Cronjob successfully exited."
+    set +x
+}
+
+trap wait_for_cron SIGINT SIGTERM
+
 while true; do
     php -f /var/www/html/cron.php &
-    sleep 5m
+    sleep 5m &
+    wait $!
```

Comment:
This line seems not necessary. Worse, if a cron job takes a long time, it will block others from getting executed

Reply:
I think this wait only waits for the sleep to be finished. We need this as otherwise the wait_for_cron is only executed after the last sleep 5m was finished...

Reply:
See https://stackoverflow.com/a/33375971